index.html

```diff
@@ -14,24 +14,36 @@
     <link rel="stylesheet" href="style.css" />
   </head>
   <body>
-    <header class="head">
+    <header class="head" id="head-mobile">
       <h3><a href="index.html">My Logo</a></h3>
-        <img id="union" src="images/Union.png" />
       <nav>
-        <ul class="links">
-          <li><a href="#">Portfolio</a></li>
-          <li><a href="#">About</a></li>
-          <li><a href="#">Contact</a></li>
+        <ul class="nav-mobile">
+          <li><a href="#grid-container" class="link">Portfolio</a></li>
```

Comment:
- [ ] Good job Team 😄 , however,  it could be better to stick to the design as much as possible (`margins` and `font-family`) as it is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md#:~:text=You%20must%20stick%20to%20the%20design%20as%20much%20as%20possible%20(e.g.%2C%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20templates%20in%20Figma.) 👍 

> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | |![Screenshot 2023-05-15 171600](https://github.com/cosywasswa/My-Portfolio-Desktop/assets/57408419/2db86b49-9d1d-4b8a-ae4e-f24fbe07d180)| |![Screenshot 2023-05-15 171657](https://github.com/cosywasswa/My-Portfolio-Desktop/assets/57408419/3c89f81a-0891-45e1-b784-07855047b8cf)| |

Kindly, Fix it 👍.